system_strings/sysmes_text.en

```diff
@@ -1147,33 +1147,33 @@ I leave the sleeping Arcueid and^exit the apartment alone, heading^for the depar
-消灯時間になった。^アルクェイドを待たせている。^部屋を出て、約束の公園に。
-吸血鬼のコミュニティ。^そのアパートは、^住宅地のただ中にあった。
-公園に戻って体を休める。^直死の魔眼とは何かを^語るアルクェイド。
-施錠のシステムが変わったのか、^屋敷の正門は閉ざされていた。^裏口を探して中に入ろう……。
-屋敷への帰路につく。^一つの歯車が、^音も無く。
-どう見てもノエル先生、だが、^どこかおかしい。^思わず身構えるが―――
-長い夜、体力の限界。^部屋まで意識が持たず、^路上で意識を失った。
-敵意も殺意もあからさまだ。^彼女は敵だと認識する。^あえて誘い込み、そして……。
+I wake up from a nostalgic^dream. The usual morning, the^usual time. But I can't find^Hisui.
+I recall last night's incident.^Akiha is waiting in the living^room. I'm unable to envision a^future where I emerge from this^alive.
+I've got to dispel this tense^atmosphere. I can't approach^this normally, so I'll proceed^candidly.
+I can't say something impudent^here. I should gently address^the situation while showing my^willingness to cooperate and^repent.
+There's no way I can talk my way^out of this. I'll go with a^full-speed, full-throttle^apology, and vigorously cut her^off.
+If I sit there, I will certainly^be executed. Resistance is^futile. I resign myself and^begin to talk.
+This laid-back guy just started^talking to me. Is this guy^really a police officer...?
+I can't deal with them^effectively if I don't even know^what it is they're after. I^should find out more about their^intentions.
+There's too much I don't^understand about the events of^the past few days. I should^avoid engaging them unprepared^and get away from them...
+I can't concentrate on the^lesson at all. Suddenly, when I^shift my gaze to the outside of^the window, there stands...
+I head to the courtyard at full^speed. I need to lure her away^before she gets the chance to^cause any mischief...!
+There's nothing I can do. I'll^concentrate on the lesson. I^need a miracle to happen right^now.
+I didn't see anything. I'll just^pretend to sleep and wait for it^to pass. 20 minutes until the^class ends...!
+Once lunch break comes around,^Arihiko shows up. I don't think^there's anything unusual going^on in town, but I should at^least ask.
+I'm curious about the abandoned^hospital Arihiko told me about.^I'm free until nighttime, so I^might as well go check it out^for a bit. 
+I'm curious about^what Arihiko said, but I doubt^it's related to the current^case. I'll head back to the^mansion and preserve my^strength.
+I return to the mansion before^sunset. I wanted to talk to^Akiha, but she hasn't returned^home yet.
+There's no way I'd be able to^deceive Hisui. I'll at least be^upfront about me leaving the^house at night.
+There's no way I could tell her^the truth. I feel bad for Hisui,^but I'll have to deceive her.
+It's lights-out time. I'm^keeping Arcueid waiting. I leave^my room and head to the park^where we promised to meet.
+A community of vampires. That^apartment complex was right in^the middle of the residential^area.
+We've returned to the park and^I'm resting my body. Arcueid^tells me about the Mystic Eyes^of Death Perception.
+It seems like they changed the^lock system. The mansion's main^gate has been locked. I'll have^to search for the back door and^enter from there...
```

Comment:
"Changed the locks" is more of a set phrase. This is also a bit repetitive, maybe modulate
> I can't get past the mansion's main gate, I wonder if they changed the locks on me. I'll have to ...